windows/src/engine/keyman32/keyboardoptions.cpp

```diff
@@ -19,8 +19,8 @@
 */
 #include "pch.h"
 
-BOOL IntLoadKeyboardOptionsCore(LPCSTR key, LPINTKEYBOARDINFO kp, km_kbp_state* const state);
-void IntSaveKeyboardOptionREGCore(LPCSTR REGKey, LPINTKEYBOARDINFO kp, LPCWSTR key, LPWSTR value);
+BOOL IntLoadKeyboardOptionsRegistrytoCore(LPCSTR key, LPINTKEYBOARDINFO kp, km_kbp_state* const state);
+void IntSaveKeyboardOptionCoretoRegistry(LPCSTR REGKey, LPINTKEYBOARDINFO kp, LPCWSTR key, LPWSTR value);
```

Comment:
Do you want to add an `s` to make the names between loading and saving consistent? (Similar in the other method names)

```suggestion
void IntSaveKeyboardOptionsCoretoRegistry(LPCSTR REGKey, LPINTKEYBOARDINFO kp, LPCWSTR key, LPWSTR value);
```

Reply:
Loading and Saving are not exactly symmetrical. The Loading works on a list of options whereas saving saves only a single option, it is a deliberate option to leave the `s` off.